client/state/data-layer/analytics/index.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * Internal dependencies
+ */
+import { mergeHandlers } from 'state/data-layer/utils';
+import adTracking from './adTracking';
+
+export const handlers = mergeHandlers(
+	adTracking
+);
+
+export default handlers;
```

Comment:
there's probably no need to make this a separately exported variable since the file is mainly here as an aggregator.

```js
export default mergeHandlers(
	adTracking,
);
```